spec/models/reservation_spec.rb

```diff
@@ -308,20 +311,30 @@
   end
 
   describe "#canceled?" do
+    # TODO: move to order_detail_spec
```

Comment:
I figure I'll leave these tests here, since Reservation#canceled? is technically still a method, and add some separately for `OrderDetail#is_canceled?`.